contracts/SnappAuction.sol

```diff
@@ -265,4 +268,46 @@ contract SnappAuction is SnappBase {
             appliedAccountStateIndex: 0
         });
     }
+
+    function orderBatchUpdate() private {
+        if (
+            auctionIndex == MAX_UINT ||
+            auctions[auctionIndex].size == maxUnreservedOrderCount() ||
+            block.timestamp > (auctions[auctionIndex].creationTimestamp + 3 minutes)
+        ) {
+            createNewPendingBatch();
+        }
+    }
+
+    function unpackAndVerifyOrderData(bytes memory packedOrder) private view {
+        // TODO - use ECRecover for signatures from first 16 bits of packedOrder
+        uint16 accountId = publicKeyToAccountMap(msg.sender);
+        // uint16 accountId = BytesLib.toUint16(orderData, 0);
+        // WARNING - currently this enables any registered member to place an order for any other registered member!
```

Comment:
For now, I'd just make sure it's msg.sender until we introduce proper signature checking.

Reply:
Do you mean require that we should fetch the accountId of msg.sender and ensure that they are registered? Meaning use the modifier `onlyRegistered()`

Reply:
Can you tell me how you are able to comment on multiple lines!

Reply:
> Can you tell me how you are able to comment on multiple lines!

In solidity or in github? I don't think I can comment on multiple lines in github. In solidity using `/* ... */`

> Do you mean require that we should fetch the accountId of msg.sender and ensure ...

yes, and ensure that accountId is that value. In fact looking at the code this is what is happening becuase the other line is commented out. Is the comment still relevant?

Reply:
Im trying to see how you are able to comment on 4 lines of code here at the once?

<img width="684" alt="Screenshot 2019-06-08 at 21 40 55" src="https://user-images.githubusercontent.com/11778116/59151613-401cb100-8a36-11e9-8b63-7143953cea07.png">